.github/CODEOWNERS

```
@@ -3,7 +3,7 @@
 # These owners will be the default owners for everything in the
 # repo. Unless a later match takes precedence, these owners will be
 # requested for review when someone opens a pull request.
-* @dav3r @felddy @jsf9k @mcdonnnj
+* @dav3r @felddy @itsmostafa @izzy64 @jsf9k @mcdonnnj
```

In this case, I think you can remove all members of the Fusion Dev team since we are not involved in this code at all:
```suggestion
* @itsmostafa @izzy64
```

---

@dav3r resolved with [e95a201](https://github.com/cisagov/con-pca-tasks/pull/1/commits/e95a2016c52ef83e86c22f499deb2cc1dcbd8dd9)